x-pack/plugins/security_solution/public/common/containers/first_last_seen/first_last_seen.test.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react-hooks';
+
+import { Direction } from '../../../../common/search_strategy';
+import { FirstLastSeenProps } from '../../components/first_last_seen';
+import { useKibana } from '../../lib/kibana';
+import { useFirstLastSeen } from '.';
+
+jest.mock('../../lib/kibana');
+
+const mockSearchStrategy = jest.fn();
+
+(useKibana as jest.Mock).mockReturnValue({
+  services: {
+    data: {
+      search: {
+        search: mockSearchStrategy.mockReturnValue({
+          unsubscribe: jest.fn(),
+          subscribe: jest.fn(({ next, error }) => {
+            try {
+              next(defaultReturn);
+              /* eslint-disable no-empty */
+            } catch (e) {}
+            return {
+              unsubscribe: jest.fn(),
+            };
+          }),
+        }),
+      },
+      query: jest.fn(),
+    },
+  },
+});
+
+const renderUseFirstLastSeen = () =>
+  renderHook<FirstLastSeenProps, ReturnType<typeof useFirstLastSeen>>(() =>
+    useFirstLastSeen({
+      order: Direction.asc,
+      field: 'host.name',
+      value: 'some-host',
+      defaultIndex: [],
+      docValueFields: [],
+    })
+  );
+
+describe('useFistLastSeen', () => {
```

Comment:
Would it be possible to add a test checking that the `search` function is called with the proper `request` data format?

Reply:
those tests have been added :D